tools/collapse_schemas/collapse-schemas.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+# The purpose of this script is a one time data migration from all of our 128
+# schemas into the public schema.
+
+set -euo pipefail
+
+usage() {
+  echo "Usage: collapse-schemas.sh <SERVER> [migrate]"
```

Comment:
Really minor - would you mind renaming this script to "merge-schemas.sh" or "migrate-schemas.sh" instead of "collapse-schemas.sh"?

Reply:
I'll rename it `merge-shards.sh`, per my comment below.

Reply:
Cool, that works!